echtvar/0.1.9/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM staphb/bcftools:1.18
+
+RUN apt update && apt install -y wget
+
+RUN wget -O /bin/echtvar https://github.com/brentp/echtvar/releases/download/v0.1.9/echtvar \
```

Comment:
Perhaps, add a version tag on the top of the file (~v0.1.9)

Reply:
Ah yeah the `ENV VERSION=0.1.9` syntax, then ${VERSION} in the wget command

Reply:
Good catch!